python/mlcroissant/mlcroissant/_src/structure_graph/nodes/metadata.py

```diff
@@ -229,7 +229,7 @@ class Metadata(Node):
     version: str | None = mlc_dataclasses.jsonld_field(
         default=None,
         description="The version of the dataset following the requirements below.",
-        input_types=[SDO.Number, SDO.Text],
+        input_types=[SDO.Integer, SDO.Number, SDO.Text],
```

Comment:
Would it make sense to have a test to check that both `"1.0"`, `1`, `1.0.0` and `1.0` are accepted?

Reply:
There's a [parametrized test](https://github.com/mlcommons/croissant/blob/e2e71a39ca106a99145f7faafe5de38c0b58cff3/python/mlcroissant/mlcroissant/_src/structure_graph/nodes/metadata_test.py#L39) about this :)